### PR TITLE
Tasks 288-289: Gigaword

### PR DESCRIPTION
**Pull request by Mihir3009**
1 Classification Task and 1 Sentence Generation Task.

Please review.

### CODE REVIEW

File: tasks/README.md [hunk 1]
```diff
@@ -227,4 +227,6 @@ Name | Summary | Category
 `task273_europarl_classification.json`| Given bulgarian sentence and corresponding english translation, verify that the translation is right or wrong | Classification
 `task274_overruling_classification`	| Given a sentence, classify it into overruling or non-overruling| Classification
 `task275_enhanced_wsc_paraphrase_generation` | Given a sentence and an aspect, paraphrase the sentence changing that aspect | Text Modification
-`task276_enhanced_wsc_classification` | Given a sentence and its paraphrase, decide what is the difference between them. | Classification
\ No newline at end of file
+`task276_enhanced_wsc_classification` | Given a sentence and its paraphrase, decide what is the difference between them. | Classification
+`task288_gigaword_sentence_generation.json` | Given a text of article, generate headline (title) for the article | Sentence Generation
+`task289_gigaword_classification.json` | Given a text of article and corresponding title, generate yes if title is appropriate, otherwise generate no | Classification
```
Comment (Palipoor) [inline tasks/README.md]: A simpler description would be:
Given the text of an article and its title, decide whether the title is appropriate for the article.

**Review by Palipoor — COMMENTED** [review]
Um, is there a reason you have  #.# s in the article bodies?

**Comment by Mihir3009** [comment]
> Um, is there a reason you have #.# s in the article bodies?

When they have to show some number like how much stocks are sold, how much tax amount is there, they hide it. I think this dataset is built on real articles, so they don't want to reveal the number I guess.

File: tasks/README.md (hunk 1 above)
Comment (Mihir3009) [reply tasks/README.md]: Thanks. Updated.

**Review by swarooprm — COMMENTED** [review]
Looks good! 
1. Create a new task category "summarization" and add these tasks there.
2. Make sure classes are not skewed in the second task.

**Comment by swarooprm** [comment]
> Looks good!
> 
> 1. Create a new task category "summarization" and add these tasks there.
> 2. Make sure classes are not skewed in the second task.

Can you address reviewer comments @Mihir3009 ?

**Comment by Mihir3009** [comment]
Resolved.
1. Added 'Summarization' Category for these tasks
2. Classes are balanced in the second file. (yes: 3250, no: 3250)

**Review by swarooprm — APPROVED** [review]
LGTM

**Pull request merged**